portsyncd/linksync.cpp

```diff
@@ -35,28 +38,79 @@ void LinkSync::onMsg(int nlmsg_type, struct nl_object *obj)
 
     struct rtnl_link *link = (struct rtnl_link *)obj;
     string key = rtnl_link_get_name(link);
-    if (nlmsg_type == RTM_DELLINK) /* Will be sync by other application */
+
+    if (key == "lo" || key == "eth0" || key == "docker0" || key == "bcm0")
```

Comment:
should those names be regexes ? like eth\d+ ? etc ? or do we only care about the ones with zeros ?